polaris-react/src/components/IndexTable/IndexTable.stories.tsx

```diff
@@ -4002,3 +4002,1435 @@ export function WithSubHeaders() {
+    let selected: IndexTableRowProps['selected'] = false;
+
+    const someProductsSelected = products.some(({id}) =>
+      selectedResources.includes(id),
+    );
+
+    const allProductsSelected = products.every(({id}) =>
+      selectedResources.includes(id),
+    );
+
+    if (allProductsSelected) {
+      selected = true;
+    } else if (someProductsSelected) {
+      selected = 'indeterminate';
+    }
+
+    const selectableRows = rows.filter(({disabled}) => !disabled);
+    const rowRange: IndexTableRowProps['subHeaderRange'] = [
+      selectableRows.findIndex((row) => row.id === products[0].id),
+      selectableRows.findIndex(
+        (row) => row.id === products[products.length - 1].id,
+      ),
+    ];
+
+    const disabled = products.every(({disabled}) => disabled);
+
+    return (
+      <Fragment key={subheaderId}>
+        <IndexTable.Row
+          rowType="subheader"
+          selectionRange={rowRange}
+          id={`Subheader-${index}`}
+          position={position}
+          selected={selected}
+          disabled={disabled}
+          accessibilityLabel={`Select all products wich has color ${color}`}
+        >
+          <IndexTable.Cell scope="col" id={subheaderId}>
+            <Text as="span" fontWeight="semibold">
+              {color}
+            </Text>
+          </IndexTable.Cell>
+          <IndexTable.Cell />
+          <IndexTable.Cell />
+        </IndexTable.Row>
+        {products.map(
+          ({id, size, quantity, price, position, disabled}, rowIndex) => {
+            return (
+              <IndexTable.Row
+                rowType="child"
+                key={rowIndex}
+                id={id}
+                position={position}
+                selected={selectedResources.includes(id)}
+                disabled={disabled}
+              >
+                <IndexTable.Cell
+                  scope="row"
+                  headers={`${columnHeadings[0].id} ${subheaderId}`}
+                >
+                  <Text variant="bodyMd" as="span">
+                    {size}
+                  </Text>
+                </IndexTable.Cell>
+                <IndexTable.Cell>
+                  <Text as="span" numeric>
+                    {price}
+                  </Text>
+                </IndexTable.Cell>
+                <IndexTable.Cell>
+                  <Text as="span" alignment="end" numeric>
+                    {quantity}
+                  </Text>
+                </IndexTable.Cell>
+              </IndexTable.Row>
+            );
+          },
+        )}
+      </Fragment>
+    );
+  });
+
+  return (
+    <LegacyCard>
+      <IndexTable
+        onSelectionChange={handleSelectionChange}
+        selectedItemsCount={
+          allResourcesSelected ? 'All' : selectedResources.length
+        }
+        resourceName={resourceName}
+        itemCount={rows.length}
+        headings={columnHeadings as IndexTableProps['headings']}
+        lastColumnSticky
```

Comment:
Seems like `lastColumnSticky` doesn't work with new Polaris version in all of the stories forIndexTable, same here.
Does anybody can confirm the same? Should I create an issue for that?